services/src/projects/project.rs

```diff
@@ -119,10 +121,28 @@ impl Project {
     }
 }
 
-#[derive(Debug, PartialEq, Serialize, Deserialize, Clone, utoipa::Component)]
+#[derive(Debug, PartialEq, Serialize, Deserialize, Clone, ToSchema)]
 #[cfg_attr(feature = "postgres", derive(ToSql, FromSql))]
 #[allow(clippy::upper_case_acronyms)]
 #[serde(rename_all = "camelCase")]
+// TODO: add example, once utoipas schema macro can co-exist with postgres OR: split type into API and database model
+// #[schema(example = json!({
+//     "boundingBox": {
+//         "lowerLeftCoordinate": {
+//             "x": -180.0,
+//             "y": -90.0
+//         },
+//         "upperRightCoordinate": {
+//             "x": 180.0,
+//             "y": 90.0
+//         }
+//     },
+//     "spatialReference": "EPSG:4326",
+//     "timeInterval": {
+//         "end": 1_388_534_400_000_i64,
+//         "start": 1_388_534_400_000_i64
+//     }
+// }))]
```

Comment:
I don't quite get it. This has nothing to do with database stuff, right?

Reply:
utoipa's dervice macro doesn't work together with the To/FromSql macros. I opened an issue at utoipa regarding this: https://github.com/juhaku/utoipa/issues/266

Reply:
can you reference the issue in the TODO?

Reply:
bdce6dee4fb6d21e20c40435a4695c572eacac14